packages/nestjs/README.md

```diff
@@ -0,0 +1,9 @@
+<p align="center">
+  <a href="https://sentry.io/?utm_source=github&utm_medium=logo" target="_blank">
+    <img src="https://sentry-brand.storage.googleapis.com/sentry-wordmark-dark-280x84.png" alt="Sentry" width="280" height="84">
+  </a>
+</p>
+
+# Official Sentry SDK for NestJS
```

Comment:
```suggestion
# Official Sentry SDK for NestJS (EXPERIMENTAL)
```

Reply:
updated the readme to indicate that it is experimental

Reply:
I would still put the change Andrei suggested. It's better to be very loud with it than people potentially breaking because they missed it.

Reply:
Yea, the Nuxt and the Solid sdk both follow this "convention" now.

Reply:
Sure, added